Spacemouse debugging

gautams3 opened the pull request:
Add a test file to debug spacemouse output

jianlanluo commented on serl_robot_infra/franka_env/spacemouse/spacemouse_test.py:
```diff
@@ -0,0 +1,21 @@
+import time
+import numpy as np
+from serl_robot_infra.franka_env.spacemouse.spacemouse_expert import SpaceMouseExpert
+np.set_printoptions(precision=3, suppress=True)
```
can you move this to test_spacemouse() ? this should be only locally valid

jianlanluo commented on serl_robot_infra/franka_env/spacemouse/spacemouse_test.py:
```diff
@@ -0,0 +1,21 @@
+np.set_printoptions(precision=3, suppress=True)
+
+def test_spacemouse():
+    """ Test the SpaceMouseExpert class. """
```
can you add detailed descriptions?

jianlanluo commented:
also can you fix the pre-commit failing ?

gautams3 commented:
1. addressed comments by @jianlanluo 
2. fixed pre-commit failures

These are a lot of commits for a minor change. It might help to squash-and-merge instead of the default.

jianlanluo commented on serl_robot_infra/franka_env/spacemouse/spacemouse_test.py:
```diff
@@ -0,0 +1,33 @@
+
+    """
+    import time
+    import numpy as np
```
import should be on top of the file

gautams3 replied:
I think I misunderstood then. You want imports to be on top, but the setting of `numpy` precision options to be in the test? both of these are global changes.

jianlanluo replied:
import should always go to the top of a py file,

np print precesion is local to your function, e.g., you can change in another function; so it should go to the function which actually needs it

gautams3 replied:
np print precision is not local. But we can make it local with a context manager. just pushed a change
